tests/test_docs/test_commands.py

```diff
@@ -158,6 +158,8 @@ def test_validate_doc_commands() -> None:
 
     # Validate all matches
     for file_ in target_files:
+        if file_.name == "upgrading.md":
+            continue
```

Comment:
This is excluded now because deprecated commands (and their replacement) are listed I suppose.